openfe/utils/visualization_3D.py

```diff
@@ -107,6 +107,45 @@ def _add_spheres(view:py3Dmol.view, mol1:Chem.Mol, mol2:Chem.Mol, mapping:Dict[i
         )
 
 
+@requires_package("py3Dmol")
+def show_component_coords(mols: Iterable[ExplicitMoleculeComponent],
+                     style: Optional[str] ="stick",
+                     shift: Optional[Tuple[float, float, float]] = None,
+                     ) -> py3Dmol.view:
+    """this function can be used to visualize multiple component coordinates in one view.
```

Comment:
This needs to be a punchy one liner, so "this function can be used to" is wasted space.  Just say "visualise multiple components in one view"

Reply:
changed